DOC.md

```diff
@@ -1064,6 +1064,20 @@ ls.add_snippets("all", {
 })
 ```
 
+- `conditions.show`: Contains typical predicats/functions used as
+  `show`-condition. Currently this is just `line_end`
+- `conditions.expand`: Contains typical predicats/functions used as
+  `expand`-condition. Currently this is just `line_begin`
+  Contains everything from `conditions.show` as well.
+- `conditions`: Provides a function `make_condition(foo)` which takes a function
+  as argument and returns a *condition object* for which several operators are
+  defined: `c1 + c2 -> c1 or c2`, `c1 * c2 -> c1 and c2`, `-c1 -> not c1`,
```

Comment:
I think this would be easier to take in as a list:
* `a or b`: `a + b`
* `a and b`: `a * b`
* `not a`: `-a`
* ...

The reasoning for `%` as `==` might also be interesting (i.e. `%` is very unexpected when one only knows that `==` can be overloaded, and not its limitations), maybe link your comment about that?

Reply:
No other reasons from my side. Trying to shortly (not sure if its short) explain why and for more details provide the link (so that you don't have to open/follow the link to understand but if you're interested you know where to look) sounds nice :+1:

Reply:
Yeah, that's alright :+1:
Throw a few newlines in there :P (alternatively, two spaces for a new paragraph)

Oh! Another thing I don't think I ever mentioned: you can use [grip](https://github.com/joeyespo/grip) to take a look at the markdown as rendered by github, locally! Makes writing doc muuuch easier, without it those issues can only be spotted after push.

Apart from that, I've no more issues, go ahead and squash,rebase

Reply:
Oh thanks, that's a really nice hint. From a first view this looks way more flexible/suited for this case then [gollum](https://github.com/gollum/gollum) (usecase is more for the wiki stuff not single `.md` files)